docs/source/reference/server-api.md

```diff
@@ -1,37 +1,29 @@
 # Starting servers with the JupyterHub API
 
-JupyterHub's [REST API][] allows launching servers on behalf of users
-without ever interacting with the JupyterHub UI.
-This allows you to build services launching Jupyter-based services for users
-without relying on the JupyterHub UI at all,
-enabling a variety of user/launch/lifecycle patterns not natively supported by JupyterHub,
-without needing to develop all the server management features of JupyterHub Spawners and/or Authenticators.
-[BinderHub][] is an example of such an application.
-
-[binderhub]: https://binderhub.readthedocs.io
-[rest api]: ../reference/rest.md
-
-This document provides an example of working with the JupyterHub API to
-manage servers for users.
-In particular, we will cover how to:
-
-1. [check status of servers](checking)
-2. [start servers](starting)
-3. [wait for servers to be ready](waiting)
-4. [communicate with servers](communicating)
-5. [stop servers](stopping)
+Sometimes, such as when working with applications such as [BinderHub](https://binderhub.readthedocs.io), it may be necessary to launch Jupyter-based services on behalf of your users. Doing so can be achieved through JupyterHub's [REST API](../reference/rest.md), which allows one to launch and manage servers on behalf of users through API calls instead of through the JupyterHub UI. In doing so, you can take advantage of other user/launch/lifecycle patterns that are not natively supported by JupyterHub, all without the need to develop the server management features of JupyterHub Spawners and/or Authenticators.
+
+This tutorial goes through the processes involved while working with the JupyterHub API to manage servers for users. In particular, it covers how to:
+
+1. [Check the status of servers](checking)
+2. [Start servers](starting)
+3. [Wait for servers to be ready](waiting)
+4. [Communicate with servers](communicating)
+5. [Stop servers](stopping)
+
+In the end, we also provide a sample Python code that can be used to implement these steps.
 
 (checking)=
 
 ## Checking server status
 
-Requesting information about a user includes a `servers` field,
-which is a dictionary.
+First, request information about a particular user using a GET request as below.
```

Comment:
I don't think we need 'following' or 'as below' when discussing things that immediately appear. A colon flows it into the sentence:

```suggestion
First, request information about a particular user with a GET request:
```